README.md

```diff
@@ -256,6 +256,7 @@ the table below for reference.
 
 | Variable | Description | Default | Required |
 | --- | --- | --- | --- |
+| AV_SCAN_PATH | Path where the file to be scanned will be temporarily stored |  | No |
```

Comment:
I think we can show the default here instead of adding a note below the table.

```suggestion
| AV_SCAN_PATH | Path where the file to be scanned will be temporarily stored | /tmp | No |
```